src/components/Waveform/Waveform.tsx

```diff
@@ -136,11 +137,13 @@ export const Waveform = forwardRef<IWaveformRef, IWaveform>((props, ref) => {
   const getAudioWaveFormForPath = async (noOfSample: number) => {
     if (!isNil(path) && !isEmpty(path)) {
       try {
+        (onWaveformLoadStateChange as Function)?.(true);
```

Comment:
@nilesh-simform Is there any specific reason of this type casting? We should remove this as the expected type has been defined already

Reply:
Yes, because the props is destructured and this prop is present in only one type, typescript considering it as never, same approach has used for other function props as well